webapp/portlet/src/main/webapp/vue-apps/attach-image/components/AttachedItems.vue

```diff
@@ -1,16 +1,22 @@
 <template>
-  <card-carousel v-if="attachmentsCount" parent-class="activity-files-parent">
-    <attached-item
-      v-for="(attachment, index) in attachments"
-      :key="attachment.id"
-      :activity="activity"
-      :index="index"
-      :count="attachmentsCount"
-      :attachment="attachment"
-      :preview-width="previewWidth"
-      :preview-height="previewHeight"
-      class="activity-file-item" />
-  </card-carousel>
+  <div>
+    <card-carousel v-if="attachmentsCount" parent-class="activity-files-parent">
+      <attached-item
+        v-for="(attachment, index) in attachments"
+        :key="attachment.id"
+        :activity="activity"
+        :index="index"
+        :attachment="attachment"
+        :preview-width="previewWidth"
+        :preview-height="previewHeight"
+        class="activity-file-item clickable"
+        @preview-attachment="openPreview(attachment.id)" />
+    </card-carousel>
+    <attachment-preview-dialog 
+      v-if="attachmentsCount"
+      ref="attachmentPreview"
+      :activity="activity" /> 
+  </div>
```

Comment:
Add `v-if="attachmentsCount"` on parent `div`

```suggestion
  <div  v-if="attachmentsCount">
    <card-carousel parent-class="activity-files-parent">
      <attached-item
        v-for="(attachment, index) in attachments"
        :key="attachment.id"
        :activity="activity"
        :index="index"
        :attachment="attachment"
        :preview-width="previewWidth"
        :preview-height="previewHeight"
        class="activity-file-item clickable"
        @preview-attachment="openPreview(attachment.id)" />
    </card-carousel>
    <attachment-preview-dialog 
      ref="attachmentPreview"
      :activity="activity" /> 
  </div>
```

Reply:
done